src/qtgui/plotter.cpp

```diff
@@ -751,12 +751,12 @@ void CPlotter::zoomStepX(float step, int x)
     qint64 new_FftCenter;
     if( limit_hit ) // cannot keep fixed_hz fixed
     {
-    	new_FftCenter = qRound64((f_min + f_max) / 2.0) - m_CenterFreq;
+        new_FftCenter = qRound64((f_min + f_max) / 2.0) - m_CenterFreq;
```

Comment:
I don't know why the tabs/spaces changed, but it's consistent now. Same as below.